### DISCUSSION TITLE
Schedule a Script

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#523] [Docs]
<meta path="help/schedule">
      
# Schedule a Script

Use cron syntax to run scripts on a schedule:

```js
// Schedule: */10 * * * * *
```

> Note: these scripts must not include `arg` or they will time out after 10 seconds

**Comment by thedjpetersen** [comment]
Does this work? I'm trying to dig through the source to figure out where it gets parsed/triggered

**Reply by johnlindquist** [reply]
Kit.app lives in a different repo.

The app watches for scripts to be added/changed then parses the script:
https://github.com/johnlindquist/kitapp/blob/main/src/watcher.ts#L59

If the script has "schedule" metadata, it will feed it into node-schedule here:
https://github.com/johnlindquist/kitapp/blob/main/src/schedule.ts#L37
